test/unit/reductions/exponentiated_gradient/test_utilities.py

```diff
@@ -5,15 +5,22 @@
 import pandas as pd
 
 sensitive_features = [str(x) for x in 'AAAAAAA' 'BBBBBBB' 'CCCCCC']
-labels =     [int(x) for x in '0110100' '0010111' '001111']  # noqa: E222
-X1 =         [int(x) for x in '0110101' '0111101' '001011']  # noqa: E222
-X2 =         [int(x) for x in '0000100' '0000011' '111111']  # noqa: E222
-X3 =         [int(x) for x in '1111111' '1111111' '111111']  # noqa: E222
+labels =             [int(x) for x in '0110100' '0010111' '001111']  # noqa: E222
+X1 =                 [int(x) for x in '0110101' '0111101' '001011']  # noqa: E222
+X2 =                 [int(x) for x in '0000100' '0000011' '111111']  # noqa: E222
+X3 =                 [int(x) for x in '1111111' '1111111' '111111']  # noqa: E222
+scores =          [int(x)/10 for x in '0001124' '0223346' '134447']  # noqa: E222
 
 
-def _get_data(A_two_dim=False):
+def _get_data(A_two_dim=False, y_as_scores=False, flip_y=False):
```

Comment:
I don't love this calling convention. The issue is that if you sat `y_as_scores` to `True`, it's not just a matter of returning `y` as real values, you are actually returning a completely different `y`. But... I don't necessarily want to be overly pedantic here... so okay to leave as is.

Reply:
Yeah, just a test utility, not an API that we expect anyone to use.